server/controllers/users/password.js

```diff
@@ -29,6 +29,7 @@ export const forgot = async function(req, res) {
   if (!user) {
     throw new ValidationError({email: 'No account with that email has been found'})
   } else if (user.provider !== 'local') {
+    if (user.provider === 'google') await mailer.sendPasswordGoogle(user)
```

Comment:
Exactly, but now it can just send this mail if user.provider is google, or save the reset token and send the regular password reset email if user.provider is local. 

Then just `res.send({message: 'Password reset email sent'})` for all cases (though it would be nice to show an error if there's a problem actually sending the email)

Reply:
Sorry I do not get it, do you want me to do something "to show an error if there's a problem actually sending the email"?
The rest it is working as you are describing.

Reply:
It's not though -
if there's no matching user it throws `ValidationError({email: 'No account with that email has been found'})`
if `user.provider = google` it sends `{message: 'It seems like you signed up using your google account'}
otherwise it sends `{message: 'Password reset email sent'}`

It should always send `{message: 'Password reset email sent'}` unless something goes wrong.